src/EditorFeatures/Test/Diagnostics/DiagnosticDataSerializerTests.cs

```diff
@@ -294,6 +294,9 @@ public class Service : IPersistentStorageService
                 IPersistentStorage IPersistentStorageService.GetStorage(Solution solution)
                     => _instance;
 
+                ValueTask<IPersistentStorage> IPersistentStorageService.GetStorageAsync(Solution solution, CancellationToken cancellationToken)
+                    => new(_instance);
```

Comment:
We also have `ValueTaskFactory` if you find it more clear:

```suggestion
                    => ValueTaskFactory.FromResult(_instance);
```

Reply:
i'm totally ok with `new` here :)